src/pudl/package_data/settings/etl_full.yml

```diff
@@ -552,11 +552,11 @@ datasets:
     tables:
       - fuel_ferc1 # requires plants_steam_ferc1 to load properly
       - plants_steam_ferc1
-      - plants_small_ferc1
+      # - plants_small_ferc1
       - plants_hydro_ferc1
       - plants_pumped_storage_ferc1
-      - plant_in_service_ferc1
-      - purchased_power_ferc1
+      # - plant_in_service_ferc1
+      # - purchased_power_ferc1
```

Comment:
all of the tables were available in the full settings but not actually etl-able so i removed them